.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ commands:
                   export TEST_AWS_ACCOUNT_ID=$(LC_ALL=C tr -dc '0-9' < /dev/urandom | fold -w 12 | head -n 1)
                   export TEST_AWS_ACCESS_KEY_ID=$TEST_AWS_ACCOUNT_ID
                   # Set TEST_AWS_REGION_NAME to a random AWS region other than us-east-1
-                  export AWS_REGIONS=("us-east-2" "us-west-1" "us-west-2" "ap-southeast-2" "ap-northeast-1" "eu-central-1" "eu-west-1")
+                  export AWS_REGIONS=("us-east-2" "us-west-1" "us-west-2" "ap-southeast-1" "ap-southeast-2" "ap-northeast-1" "eu-central-1" "eu-west-1")
```

Comment:
just in case this slips back in, this is an older version of the current file, and this should not get merged
(@sannya-singal is working hard to greenify the pipeline and this would undo her change with #10540)

Reply:
Thanks - rebased :)

Reply:
@maxhoheiser Why do we need to change the regions in this PR?

This PR is still undoing the change from @sannya-singal here: https://github.com/localstack/localstack/pull/10540